helm/values.yaml

```diff
@@ -123,10 +129,13 @@ admissionWebhooks:
 agent:
   name:
   image:
-    repository: cloudwatch-agent
-    tag: 1.300031.1b317
+    # repository: cloudwatch-agent
+    # tag: 1.300031.1b317
+    repository: tupperware
```

Comment:
Are there changes to the agent that we need to make to get this to work?

Reply:
nope. removed test repo